### ISSUE TITLE
Create a JS playground with @chat-e2ee/service package

### ISSUE THREAD

**Issue by muke1908** [muke1908/chat-e2ee#289]
Create a fiddle using @chat-e2ee/service package to demonstrate basic functionalities.

**Comment by rohan-naik07** [comment]
Hi @muke1908 .... can I take up this?

**Comment by muke1908** [comment]
@rohan-naik07 Sure, go ahead.

**Comment by rohan-naik07** [comment]
Whats exactly a fiddle?... like a web app that exposes the APIs to create dummy channels?

**Comment by rohan-naik07** [comment]
fiddle means online code editor right that exposes all the APIs of the sdk ...like two code editors which represent 2 users and we can create a dummy channel between them?

**Comment by muke1908** [comment]
@rohan-naik07 yes, working demo of the exposed APIs on the SDK.

**Comment by rohan-naik07** [comment]
So basically I will proceed with a new react app called e2ee-playground. 
- this frontend will have 2 open code editors which expose all the apis of the SDK.
- person can run the code in the editors on the cloud runtime (eg- aws lamdba) and render the logs of each editor below the each editor respectively

**Comment by muke1908** [comment]
I think you can just keep it vanilla-js, react-app is unnecessary imo. 
You can use codesandbox for writing code. 
Instead of 2 editor, we can simulate 2 users using iframe - user1 is main window, user2 embedded in an iframe. 

Here is an example: https://codesandbox.io/p/sandbox/gracious-pine-6jhqwt

**Comment by rohan-naik07** [comment]
So its just creating two html pages in the sandbox which simulate two users and embed the second user in first user's window. the two users can send messages to each user using common channel.

**Comment by muke1908** [comment]
Yes. I didn't understand what you meant by "common channel" but they both will have their own `chatInstance`

**Comment by Jatin917** [comment]
is anyone working on it?
